test/stomp_test.dart

```diff
@@ -23,7 +19,7 @@ void main() {
         import 'package:web_socket_channel/io.dart';
         import 'package:stomp_dart_client/stomp_parser.dart';
```

Comment:
Can you also please update those imports? Those are still throwing an error. There are a few in the hybrid tests.

Reply:
good catch! Should be fixed, let me know if I missed any place. As I said in the PR's description I can't easily run these tests myself..